runner/main.go

```diff
@@ -100,6 +100,7 @@ func run(ctx context.Context, args []string, stdout, stderr io.Writer) error {
 		{Bin: "node", Args: []string{"puppeteer/cdp.js"}, Env: []string{"RUNS=10"}},
 		{Bin: "node", Args: []string{"puppeteer/dump.js"}, Env: []string{"URL=http://127.0.0.1:1234/campfire-commerce/"}},
 		{Bin: "node", Args: []string{"puppeteer/links.js"}, Env: []string{"URL=http://127.0.0.1:1234/campfire-commerce/"}},
+		{Bin: "node", Args: []string{"puppeteer/click.js"}, Env: []string{"URL=http://127.0.0.1:1234/campfire-commerce/"}},
```

Comment:
`click.js` doesn't take an url as parameter.
```suggestion
		{Bin: "node", Args: []string{"puppeteer/click.js"}},
```

Reply:
Fixed: [63dc179](https://github.com/lightpanda-io/demo/pull/37/commits/63dc1791b671629496becf75511fcd4acf32e12a)